### ISSUE TITLE
[Template] Prefilled call stack, traces and configuration

### ISSUE THREAD

**Issue by AlitzelMendez** [dotnet/arcade#13456]
Feedback: 
`It would be nice if people paste the full callstack in the description of the issue, because the log files get deleted from the servers pretty fast, and having that info handy in the issue description helps everyone identify the exact problem quickly. Otherwise, they have to click on a recent build result and then having to find the log file for this failure.`

`Pre-filled template would contain failed configurations, exception message and call stack `

This information is in relation to the issue originally reported. There are a couple of **things to consider**:
- When new builds match with the known issue, this information is not going to be updated. This is important, especially for the configuration information, as it could originally match some configuration but that doesn’t mean those will be the only configurations impacted.
- Call stacks are very long in some cases. We could limit this to a fixed number of lines and put this information as a ‘suggested’ call stack. However, we can’t guarantee that the whole call stack is going to be present.

**Implementation considerations:** 
We would need to reroute all known issue creation to our Helix API (to prevent hitting our 65k limit), which will redirect to GitHub with all the information. We would need an intermediaries table to pull this information, causing a delay in the creation of the GitHub issue. The experience will be negatively impacted for this. 

Another option is to populate this information after the known issue is created. In this way, we would not impact the experience per se but will need a new workflow.

This is not a straightforward change, but it can be done. 

### Release Note Category
- [ ] Feature changes/additions 
- [ ] Bug fixes
- [ ] Internal Infrastructure Improvements

**Comment by AlitzelMendez** [comment]
I started working on this but the more I get into this, the more hesitant I am on providing this feature. The main reason is that I think this feature can’t provide the accuracy and context needed for it to be useful.

There are different places where a known issue can occur:

Call stack or AzDO Error Message section: In this particular case, we can provide all the information as it is already processed, formatted and tends to be small.
Build errors (the ones listed at the top): These don’t have any context, so even if we copy-paste the error message, it is unclear what is happening.
Step logs: We know which is the matching line, as we do the matching process by line, but we don’t know how many lines will give the perfect context for the specific error.
Helix logs: Same scenario as above.

In conclusion, we can’t analyze the logs to identify which lines are needed to provide the context. We only know the failures by line, which is not useful

**Comment by AlitzelMendez** [comment]
Moving this to revisit phase

**Comment by AlitzelMendez** [comment]
closing as currently the is not enough information to known which is the actual error in a log, which would end up with us provided useless information.
